client/packages/system/src/Asset/api/hooks/log/useAssetLogReasons.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery, useUrlQueryParams } from '@openmsupply-client/common';
+import { useAssetApi } from '../utils/useAssetApi';
+
+export const useAssetLogReasons = (storeId?: string | undefined) => {
+  const { queryParams } = useUrlQueryParams({
+    filters: [
+      { key: 'id' },
+      { key: 'assetLogStatus', condition: 'equalTo' },
+      { key: 'reason' },
+    ],
+  });
+
+  const id = storeId ?? '';
```

Comment:
I'm providing an optional storeId to pass to the back end so that it is accessible in the service_context.

But maybe it isn't required. I am making an alternative branch now without passing storeId through